skyportal/models.py

```diff
@@ -206,6 +208,69 @@ def desi_dr8_url(self):
         return (f"http://legacysurvey.org/viewer/jpeg-cutout?ra={self.ra}"
                 f"&dec={self.dec}&size=200&layer=dr8&pixscale=0.262&bands=grz")
 
+    def airmass(self, telescope, time, below_horizon=np.inf):
+        """Return the airmass of the Obj at time `time` from Telescope
+        `telescope`.
+
+        Parameters
+        ----------
+
+        telescope: skyportal.models.Telescope
+            The telescope to use for the airmass calculation
+        time: astropy.time.Time
+            The time or times at which to calculate the airmass
+
+        Returns
+        -------
+        secz: ndarray in the shape of `time`
+           The airmass of the Obj at the requested times
```

Comment:
You are no longer sending `secz` but instead the Pickering value of the Rayleigh (molecular atmosphere) airmass. Just call this variable `airmass`. 
```suggestion
        airmass: ndarray in the shape of `time`
           The airmass of the Obj at the requested times
```
You should document what it means to be below the horizon (numerically). [Looks like the limit of airmass](https://www.wolframalpha.com/input/?i=lim%281%2Fsin%28%28pi%2F180%29*%28a+%2B+244+%2F+%28165+%2B+47*a**1.1%29%29%29%2C+a-%3E0%29) as h->0 is 38.7494. Might want to document that.